micronaut-graalpy/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id "io.micronaut.build.internal.graal-languages-module"
+}
+
+// XXX GP
+// Execution failed for task ':micronaut-graalpy:findBaseline'.
+// > Could not find a previous version for 1.0.0
+findBaseline.enabled = false
+
+repositories {
+    // XXX GP
+    maven {
+        url "file:///tmp/local_repo"
+    }
+    mavenCentral()
+}
+
+dependencies {
+    api(group: "org.graalvm.python", name: "python", version: libs.versions.graalpy.get(), ext: "pom", transitive: true)
```

Comment:
- moved `version` from the previous `implementation("org.graalvm.python:python:24.2.0@pom")` to `libs.versions.toml`, but was struggling how to declare the `extension` as `pom`. Maybe there is better way than this?
- also that with `implemention` it wasn't possible to access transitive deps from a micronaut project - e.g. `org.graalvm.polyglot.Context`. Than again, not sure if `api` is the right way how to achieve that.